src/tests/frontend/cypress/cypress/integration/test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+// TODO: See if there's an ESLint environment for Cypress.
+/* global Cypress, cy */
+
+Cypress.Commands.add('iframe', {prevSubject: 'element'},
+    ($iframe) => new Cypress.Promise((resolve) => {
+      $iframe.ready(() => {
+        resolve($iframe.contents().find('body'));
+      });
+    }));
+
+describe(__filename, function () {
+  it('Pad content exists', async function () {
+    cy.visit('http://127.0.0.1:9001/p/test', {timeout: 60000});
+    cy.wait(2000);
```

Comment:
I don't think this is necessary—the docs say that Cypress automatically retries until timeout or the condition is satisfied.

Reply:
The issue is that on initial page load Etherpad is still building cache so we want to wait for it to be fully built...  

HTTP connectivity !== Pad Availability.  I check for Pad Availability.

Reply:
I can drop them and see, but I am pretty sure the wait is required.  I assume you are referring to the timeouts not being necessary?

Reply:
I'm referring to the `cy.wait()` calls. See:
https://docs.cypress.io/guides/references/best-practices.html#Unnecessary-Waiting